src/services/Task/TaskReview/TaskReview.js

```diff
@@ -256,7 +257,12 @@ export const setupFilterSearchParameters = (filters, boundingBox, savedChallenge
     searchParameters.endDate = format(filters.reviewedAt, 'YYYY-MM-DD')
   }
   if (filters.challengeId) {
-    searchParameters.cid = [filters.challengeId]
+    if (!_isArray(filters.challengeId)) {
+      searchParameters.cid = filters.challengeId
+    }
+    else {
+      searchParameters.cid = _join(filters.challengeId, ',')
```

Comment:
You don't need a lodash _join for this. You can just do `filters.challengeId.join(',')` since you've already ensured filters.challengeId is an array.